packages/blade/src/components/BladeProvider/__tests__/paymentLightTheme/paymentLightTheme.native.ts

```diff
@@ -0,0 +1,31 @@
+import { Easing } from 'react-native-reanimated';
+
+import type { Theme } from '../../';
+import paymentLightThemeWeb from './paymentLightTheme.web';
+
+const paymentLightTheme: Theme = {
+  ...paymentLightThemeWeb,
+  motion: {
+    ...paymentLightThemeWeb.motion,
+    easing: {
+      standard: {
+        attentive: Easing.bezier(0.5, 0, 0.3, 1.5),
```

Comment:
Since this was sort of a 'mock' theme, I wanted to put in raw values. Especially because we use this theme to do the check `expect(result.current.theme).toEqual(paymentLightTheme);` - here we should have our "expectations" as raw as possible instead of making it dependent on another function that can change over time leading to our "expectations" changing over time. So in case, makeBezier evolves & changes, it should force the person making the change to relook at these tests and change it here as well. Thoughts?

Reply:
makes complete sense 👍